h2o-extensions/target-encoder/src/main/java/hex/schemas/TargetEncoderModelOutputV3.java

```diff
@@ -1,14 +1,7 @@
 package hex.schemas;
 
 import ai.h2o.targetencoding.TargetEncoderModel;
-import water.api.API;
 import water.api.schemas3.ModelOutputSchemaV3;
 
-public class TargetEncoderModelOutputV3 extends ModelOutputSchemaV3<TargetEncoderModel.TargetEncoderOutput, TargetEncoderModelOutputV3> {
-  @API(help = "Prior mean value", direction = API.Direction.OUTPUT)
-  public double prior_mean;
```

Comment:
a single priorMean doesn't make any sense in case of multiclass.
As it seemed to have been mainly exposed only for testing purposes on the client side, I prefer to completely remove it.